histomicstk/deeplab/utils/wsi_dataset_util_large_image.py

```diff
@@ -20,6 +20,22 @@
 ################################ main function #################################
 ################################################################################
 
+def get_image(filename):
+    """
+    Read the inut image and perform channel decomposition if necessary.
+    """
+
+    slide = large_image.open(filename)

```

Comment:
@sayatmimar: don't we need to pass any other arguments in `open` method like `r` or `w`

Reply:
Not needed per documentation, this function can read the file.